### ISSUE TITLE
Bank Reconciliation Tool: Impossible to know which transaction comes first for matching

### ISSUE THREAD

**Issue by nissekalle0** [frappe/erpnext#36448]
### Information about bug

![Screenshot from 2023-08-01 12-32-44](https://github.com/frappe/erpnext/assets/88166895/ce66e2c3-1394-432f-9dae-d4eefc9b2207)

Which one of the 5 items is first to match with which one? The info in the list is not good enough and the risk is high that transactions are mixed...when matching!

Solution: More transaction info for each item.

### Module

accounts

### Version

Latest

### Installation method

manual install

### Relevant log output / Stack trace / Full Error Message.

```shell
No errors
```

**Comment by bosue** [comment]
Why is there no party given? Why are both amount and reference number the same for several items?
Please elaborate a bit on why this is a valid usecase at all, even if it might be an edge case, and which information that does exist but isn‘t displayed would help you making the right choice. Thanks!

**Comment by nissekalle0** [comment]
Party is not given automatically by the immport. The bank has same ref num. It seems to be a type. Amount is the same because it is a line amortization. To pay back loans is a valid use case! No other info as far as I can see..

**Comment by bosue** [comment]
Of course, loan repayment is a valid use case. I don't want to annoy you, but you seriously need to be a bit more specific:

- If parties aren't given, how can we know whether they are the same or different? Now let's take for granted that it will be the same, as per your explanation. But the following question remains relevant: Is the party info not given in the import file or is it dismissed during import, because it can't be matched?
- Same amount, same payee line and same ref number is quite unfortunate. Besides these, what kind of information is in the import file that could be imported and displayed, but isn't?

See why I'm asking? I'm willing to help, but need more info about what _may_ help.

**Comment by DipenFrappe** [comment]
Replicate the issue in the latest version, and if it replicates, help with steps and screenshots/GIFs
